### PR TITLE
fixed small case matching bug

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Signed-off-by: Sam Markowitz <sam.markowitz@hpe.com>

### CODE REVIEW

File: content/io/cloudslang/base/lists/find_all.sl [hunk 1]
```diff
@@ -16,9 +16,16 @@ operation:
   inputs:
     - list
     - element
+    - ignore_case:
```
Comment (tethryus) [inline content/io/cloudslang/base/lists/find_all.sl]: @SamMarkowitz Please add the new input (ignore_case) in the description.

**Comment by tethryus** [comment]
@SamMarkowitz In the description for the operation find_all.sl the input: element is defined as @output element.

**Comment by SamMarkowitz** [comment]
@tethryus, thanks for the catch. Fixed.

**Comment by SamMarkowitz** [comment]
@tethryus, I added some tests.

**Comment by tethryus** [comment]
@SamMarkowitz Everything looks ok, ready for merge.

**Pull request merged**